module/VuFind/src/VuFind/ILS/Driver/CacheTrait.php

```diff
@@ -91,7 +91,8 @@ protected function getCachedData($key)
         $item = $this->cache->getItem($fullKey);
         if (null !== $item) {
             // Return value if still valid:
-            if (time() - $item['time'] < $this->cacheLifetime) {
+            $lifetime = $item['lifetime'] ?? $this->cacheLifetime;
```

Comment:
I think the changes to this file stand alone and would probably make sense as a commit outside of this PR. Do you have any objections to me cherry-picking these changes to master now?

Reply:
No objections. Thanks!